coso/.htaccess

```diff
@@ -0,0 +1,40 @@
+# # /coso/
+#
+# http://w3id.org/coso/ html request redirects to 
+# https://gdb.acg.maine.edu:7201/resource?uri=http:%2F%2Fw3id.org%2Fcoso%2Fv1%2Fcontaminoso%23 for html request
+# to https://raw.githubusercontent.com/SAWGraph/contaminoso/refs/heads/main/v1/contaminoso_core.ttl for base request
+#
+# http://w3id.org/coso/ ttl request redirects to
+#
+# 
+# ## Contact
+# This space is administered by:
+#
+# Katrina Schweikert
+# Github username: kschweikert
+#
+# Torsten Hahmann
+# Github username: thahmann
+
+RewriteEngine on
+
+# Rewrite rule to serve directed HTML content from class/prop URIs
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml
+RewriteRule ^v1/contaminoso#(.+) https://theskailab.github.io/COSO/contaminoso.html#$1 [B]
```

Comment:
Browsers are not likely to send along hash fragments.  Unless you are dealing with some special case clients that do send those, you can leave that part off and the browser will probably do what you want.  Same below.  That probably removes the need for `[B]`.